### ISSUE TITLE
Aria-valuetext for sliders do not update when Solution Values checkbox is checked/unchecked 

### ISSUE THREAD

**Issue by Matthew-phet** [phetsims/molarity#132]
Aria value text for sliders do not update immediately when checked box is checked/unchecked, the sliders have to be refocused on for them to update, which leads to a weird mismatch of qualitative and quantitative aria value text. 
![image](https://user-images.githubusercontent.com/48224214/61752031-2eca0280-ad5f-11e9-92f3-382d603200ca.png)

**Comment by terracoda** [comment]
@Matthew-phet, thanks for posting. I'll add another screen shot that includes the visuals:

<img width="756" alt="Screen Shot 2019-07-23 at 5 56 31 PM" src="https://user-images.githubusercontent.com/1902116/61752308-7d04f300-adab-11e9-951f-45746b8d3be7.png">

**Comment by terracoda** [comment]
@zepumph, did you do anything useful in GFLB that would be relevant to this issue? I know you linked the two move-sphere sliders, but I'm not sure that would generalizable to this case?

**Comment by zepumph** [comment]
@twant I believe this is the issue we discussed this afternoon. In general I think that https://github.com/phetsims/molarity/commit/b0c89d2d8a6a979c959ebae855e94ff176067299 is a hack, and should not be implemented. Ideally AccessibleValueHandler should be the only thing directly setting aria-valuetext. The main reason for that is that I would hate for overwrite conditions or convoluted Property dependency order to produce weird logic as it came to the order in which aria-valuetext is set. If you can't see an element of AccessibleValueHandler that reaches your needs, then let me know and we can discuss!

**Comment by twant** [comment]
Thanks for working through this with me, @zepumph! Assigning back over to you for review, @Matthew-phet and @terracoda!

**Comment by terracoda** [comment]
Awesome work @twant and @zepumph. Looks good on my end. Closing.
